inc/datahandlers/post.php

```diff
@@ -708,14 +708,7 @@ function verify_prefix()
 						}
 					}
 
-					if($required['groups'] != "-1")
-					{
-						if(!is_member($required['groups'], array('usergroup' => $user['usergroup'], 'additionalgroups' => $user['additionalgroups'])))
-						{
-							$num_prefixes = true;
-						}
-					}
-					else
+					if(is_member($required['groups'], array('usergroup' => $user['usergroup'], 'additionalgroups' => $user['additionalgroups'])))
```

Comment:
As I've already mentioned elsewhere: Why not pass `$user` directly? (IIRC I've already change it here in one of my PRs)